### PR TITLE
Adding FAQ chapter 

### PR DESCRIPTION
**Pull request by williln**
Also, an accessibility header correction that I think has already been merged got in here somehow? Let me know if I need to do something... I am new to git.

### CODE REVIEW

File: faq/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# FAQ: Frequently Asked Questions 
+
+Django Girls events have been happening all over the world for awhile now, and we get some repeat questions. Hopefully this page will help you if part of the planning process has you confused!
```
Comment (bmispelon) [inline faq/README.md]: I thought the expression was "happening for a while", not "awhile". Am I mistaken?

**Comment by williln** [comment]
@bmispelon You are correct, sir! Thanks for that catch.

**Comment by bmispelon** [comment]
Looks good to me.

I'll try to do some git magic to merge it. Make sure you're wearing your :eyeglasses:

**Comment by williln** [comment]
puts on :sunglasses:

**Comment by bmispelon** [comment]
No idea what was wrong but git wouldn't let me apply your patch.

I ended up doing it semi-manually and I think I got it to work (370352e3058d3de7e46e8c93457962e6c996be87)...

:computer: :fire:

**Pull request closed without merging**